lib/index.js

```diff
@@ -251,11 +247,10 @@ api.get = brCallbackify(async ({actor, id}) => {
     resource: [id]
   });
 
-  const record = await database.collections.account.findOne(
-    {id: database.hash(id)}, {_id: 0, account: 1, meta: 1});
+  const record = await database.collections.account
+    .findOne({id: database.hash(id)}, {_id: 0, account: 1, meta: 1});
   if(!record) {
-    throw new BedrockError(
-      'Account not found.',
+    throw new BedrockError('Account not found.',
```

Comment:
If one parameter is indented, all params should be indented (the way it was), this is happening now in several places in the commit.  Revert style of previous code.

Reply:
Yeah, needs reverting. Looks like we still don't have the eslint rules quite right.

Reply:
yeah this was caused by never with "consistent" if the first paren has a new line the last paren must have a new line. What you want is if paramaters exceed line length then allow a new line otherwise do not.